frontend/public/components/operator-lifecycle-manager/index.tsx

```diff
@@ -218,6 +218,7 @@ export type PackageManifestKind = {
       name: string;
       currentCSV: string;
       currentCSVDesc: {
+        description?: string;
```

Comment:
Why this change?

Reply:
FWIW, `oc explain packagemanifests.status.channels.currentCSVDesc` shows all of these fields as optional. This might be a mistake in the schema, though.

Reply:
The missing description property was causing a typescript error.